src/components/PetForm.jsx

```diff
@@ -81,7 +87,15 @@ export default function PetForm({ initialValues, onSubmit, type }) {
             type="submit"
           >
             {type}
-          </SubmitButton>
+          </PrimaryButton>
+          <SecondaryButton
+            type="button"
+            onClick={() => {
+              navigate(-1);
```

Comment:
This can have unexpected behaviour, if you were come to the form via a hard link (you wrote the url in the browser) it would take you back to the previous page you were on instead of the list, it should just navigate to the url of the list instead of the previous page